client/src/main/java/com/tigrisdata/db/client/StandardTigrisAsyncDatabase.java

```diff
@@ -171,6 +177,33 @@ public CompletableFuture<DatabaseDescription> describe() throws TigrisException
         DESCRIBE_DB_FAILED);
   }
 
+  @Override
+  public void stream(TigrisAsyncStreamer streamer) {
```

Comment:
nit: In my mind `streamer` is the one which originates the stream. This is more of a StreamCallback or StreamListener - wdyt?

Reply:
I was just following the pattern from the read method, let's discuss tomorrow